feature-group-definitions/counter-style.yml

```diff
@@ -0,0 +1,28 @@
+name: "@counter-style"
+description: The `@counter-style` CSS at-rule defines custom counter styles for list items. For example, you can use a sequence of specific symbols instead of numbers for an ordered list.
```

Comment:
`@font-face` and `@scope` don't have descriptions yet, but I think "CSS at-rule" will work in general 👍